File: app/src/main/java/bassamalim/hidaya/database/TelawatRecitersDB.java
package bassamalim.hidaya.database;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "telawat_reciters")
public class TelawatRecitersDB {

    @PrimaryKey
    @ColumnInfo(name = "reciter_id")
    private final int reciter_id;
    @ColumnInfo(name = "reciter_name")
    private final String reciter_name;

    public TelawatRecitersDB(int reciter_id, String reciter_name) {
        this.reciter_id = reciter_id;
        this.reciter_name = reciter_name;
    }

    public int getReciter_id() {
        return reciter_id;
    }

    public String getReciter_name() {
        return reciter_name;
    }
}

File: app/src/main/java/bassamalim/hidaya/core/data/database/dbs/TelawatRecitersDB.kt
package bassamalim.hidaya.core.data.database.dbs

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "telawat_reciters")
data class TelawatRecitersDB(
    @field:ColumnInfo(name = "id") @field:PrimaryKey val id: Int,
    @field:ColumnInfo(name = "name_ar") val name: String,
    @field:ColumnInfo(name = "name_en") val nameEn: String,
    @field:ColumnInfo(name = "favorite") val favorite: Int
)